### ISSUE TITLE
obcommand app:reload cause system to crash and reboot

### ISSUE THREAD

**Issue by twio142** [esm7/obsidian-vimrc-support#87]
This is the command I put in my vimrc file: `exmap R obcommand app:reload`

Every time I type `:R` to reload the app, it cause my mac to crash and reboot...

**Comment by esm7** [comment]
Does the command work ok when you launch it normally through Obsidian ("Reload app without saving" in the command palette)?

**Comment by twio142** [comment]
@esm7 Yes, reload through the command palette works properly. Even typing `:obcommand app:reload` doesn't cause the crash.
